__tests__/app.test.js

```diff
@@ -72,17 +73,24 @@ describe("GET /api/articles", () => {
         expect(response.body.msg).toBe('Bad request');
       });
   });
-  /*
+  
   test("GET:200 sends an array of articles to the client", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body: { articles } }) => {
-        console.log(articles)
-        //expect(articles).toEqual(endpointsJson);
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
```

Comment:
Well done for asserting against the length here to ensure the test does not pass by default if we get an empty array back! :smile: